Core/Common/PaginationHelper.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Core.Interfaces;
+using Core.Response;
+using Core.Services;
+using System;
+using System.Collections.Generic;
+
+namespace Core.Common
+{
+    public class PaginationHelper
+    {
+
+        public static PagedResponse<T> CreatePagedReponse<T>(T pagedData, PaginationFilter validFilter, int totalRecords, IUriService uriService, string route)
+        {
+
+            var respose = new PagedResponse<T>(pagedData, validFilter.PageNumber, validFilter.PageSize);
+            var totalPages = validFilter.PageSize == 0 ? 1 : ((double)totalRecords / (double)validFilter.PageSize);
+            int roundedTotalPages = Convert.ToInt32(Math.Ceiling(totalPages));
```

Comment:
You can shorten it to 
`var totalPages = validFilter.PageSize == 0 ? 1 : (int)Math.Ceiling((double)totalRecords / validFilter.PageSize);`

Reply:
Okey, will be done